.github/workflows/test.yml

```diff
@@ -20,6 +20,7 @@ jobs:
             vhpidirect/quickstart/random,
             vhpidirect/quickstart/math,
             vhpidirect/quickstart/customc,
+            vhpidirect/quickstart/logicvector,
```

Comment:
You need to add it twice now, one for Linux and another one for Windows (below).

Reply:
okidoki